validmind/api_client.py

```diff
@@ -555,6 +555,66 @@ def log_input(name: str, type: str, metadata: Dict[str, Any]) -> Dict[str, Any]:
         raise e
 
 
+async def alog_metric(
+    key: str,
+    value: float,
+    inputs: Optional[List[str]] = None,
+    params: Optional[Dict[str, Any]] = None,
+    recorded_at: Optional[str] = None,
```

Comment:
Where/who do you suggest sets this value?

Reply:
I added this here since its an optional parameter that the backend accepts and is a distinct field from created_at. Reason being is I figured we might have cases where monitoring data is computed live and then batch-shipped off to the platform. For now I was leaving it here so we are aware that that option exists but happy to remove it if we feel it's not necessary.